pkgs/development/python-modules/pysmart-smartx/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy3k
+, future
+, pytest
+, mock
+}:
+
+buildPythonPackage rec {
+  pname = "pysmart-smartx";
+  version = "0.3.10";
+
+  src = fetchFromGitHub {
+    owner = "smartxworks";
+    repo = "pySMART";
+    rev = "v${version}";
+    sha256 = "1irl4nlgz3ds3aikraa9928gzn6hz8chfh7jnpmq2q7d2vqbdrjs";
+  };
+
+  propagatedBuildInputs = [ future ];
+
+  # tests require contextlib.nested
+  doCheck = !isPy3k;
+
+  checkInputs = [ pytest mock ];
+
+  checkPhase = ''
+    py.test $src/pySMART/test
```

```suggestion
    py.test test
```

Does pytest not find the test directory automatically? Also this could use pytestCheckHook.

---

If I just do `py.test test` it does not work because the file is located in the `$src`, but `pytestCheckHook` works.